.github/workflows/tests.yml

```diff
@@ -39,10 +39,8 @@ jobs:
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v3
       with:
-        dotnet-version: "3.1.x"
-    - name: Install dotnet-format
-      run: dotnet tool install -g dotnet-format
```

Comment:
in .net 6 you can do `dotnet format .`

Reply:
Thanks, @ahmednfwela !
Perhaps it would be worthwhile to rewrite this section `### Style Rules` of the contribution.
In this case, @alrz1999 you can do it if you want or ignore my last comment about `dotnet format --check Meilisearch.sln` and I will open an issue about this rewriting.